nextroute/engine.go

```diff
@@ -179,4 +179,10 @@ var (
 	newTimeIndependentDurationExpression func(
 		DurationExpression,
 	) TimeDependentDurationExpression
+	solve func(
```

Comment:
Too much sugar, we don't need to hide any of the code invoked in this method (better yet we might need it to change some things)